Rework etl exercise (#24)
kytrinyx/problem-specifications: exercises/etl/instructions.md

# Instructions

Your task is to change the data format of letters and their point values in the game.

Currently, letters are stored as a list, per score.

- 1 point: "A", "E", "I", "O", "U", "L", "N", "R", "S", "T",
- 2 points: "D", "G",
- 3 points: "B", "C", "M", "P",
- 4 points: "F", "H", "V", "W", "Y",
- 5 points: "K",
- 8 points: "J", "X",
- 10 points: "Q", "Z",

This needs to be changed to store each letter with its score.
As part of this change, the team has also decided to change the letters to be lower-case rather than upper-case.

- "a" is worth 1 point.
- "b" is worth 3 points.
- "c" is worth 3 points.
- "d" is worth 2 points.

> - etc.

Author: This is the same input that it used to be; not really sure how to change this without using actual programming syntax... and the syntax varies by language.

> - etc.

Author: Ok, I've tried to add some wording to clarify, and added a note to tell people to look at the test suite, since we can't specify an actual code sample in the instructions.

# Instructions

> Your task is to change the data format of letters and their point values in the game.

Reviewer: This is much more how I think the task should be stated.

Currently, letters are stored in groups based on their score, in a one-to-many mapping.

- 1 point: "A", "E", "I", "O", "U", "L", "N", "R", "S", "T",
- 2 points: "D", "G",
- 3 points: "B", "C", "M", "P",
- 4 points: "F", "H", "V", "W", "Y",
- 5 points: "K",
- 8 points: "J", "X",
- 10 points: "Q", "Z",

This needs to be changed to store each individual letter with its score in a one-to-one mapping.

- "a" is worth 1 point.
- "b" is worth 3 points.
- "c" is worth 3 points.
- "d" is worth 2 points.
- etc.

As part of this change, the team has also decided to change the letters to be lower-case rather than upper-case.

~~~~exercism/note

> If you want to look at how the data used to be structured and how it needs to change, take a look at the examples in the test suite.

Reviewer: ```suggestion
If you want to look at how the data was previously structured and how it needs to change, take a look at the examples in the test suite.
```

Reviewer: I misread "used" the first time.
